src/pages/components/breadcrumb/accessibility.mdx

```diff
@@ -6,335 +6,53 @@ description:
 </AnchorLinks>
 
-## How it works
+## What Carbon provides
 
-The `Breadcrumb` component implements the
-[Breadcrumb design pattern from WAI ARIA](https://www.w3.org/TR/wai-aria-practices-1.1/#breadcrumb)
-for Carbon. It establishes a landmark on the page which assists the user in
-understanding where they currently are and which pages exist in the current
-page's hierarchical order.
+Carbon bakes keyboard operation into its components, improving the experience of blind users and others who operate via the keyboard. Carbon incorporates many other accessibility considerations, some of which are described below.
 
-The landmark is provided through the `<nav>` element which contains a list of
-links. The `<nav>` element is labeled by the `aria-label`attribute, providing a
-chance to describe to the user the type of navigation provided by the list of
-links. The current page is identified with the `aria-current="page"` attribute
-on the appropriate link.
+### Keyboard interactions
 
-## Accessibility considerations
+Each page link in the breadcrumb is reached by `Tab` and activated by `Enter`. The current page, if listed in the breadcrumb, is not a link. If the breadcrumb is truncated, the ellipsis button for the overflow menu is in the tab order. See [overflow menu](https://carbondesignsystem.com/components/overflow-menu/usage/) for details on its keyboard operation.
 
-This component has been validated to meet the
-[WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) and
-[Section 508](http://www.section508.gov/) accessibility guidelines, however
-changes made by the content owner can affect accessibility compliance. Be sure
-to review and follow the guidance in this section when updating or adding new
-content to this component.
-
-1. The breadcrumb should remain in the same relative order and location on each
-   web page.
-1. The link text should be clear, reflect the breadcrumb location accurately and
-   easy to understand for all users.
-1. The link to the current page element in the breadcrumb has the
-   `aria-current="page"` attribute.
+<Row>
+<Column colLg={8}>
 
-## Resources
+![example of breadcrumb keyboard interaction](images/breadcrumb-accessibility-1.png)
 
-- [W3C WAI-ARIA Authoring Practices Breadcrumb Design Pattern](https://www.w3.org/TR/wai-aria-practices/#breadcrumb)
-  covers the usage of ARIA names, state and roles.
-- [IBM Accessibility Requirements:](https://www.ibm.com/able/requirements/requirements/)
-  - [2.4.4 Link Purpose (In Context)](https://www.ibm.com/able/requirements/requirements/#2_4_4)
-    (WCAG Success Criteria
-    [2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html))
-  - [3.2.3 Consistent Navigation](https://www.ibm.com/able/requirements/requirements/#3_2_3)
-    (WCAG Success
-    Criteria[3.2.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-locations.html))
+<Caption>The breadcrumb's links are reached by `Tab` and activated by `Enter`.</Caption>
```

Comment:
We can take the code styling off of Tab and Enter in the caption, because the caption does not support this styling.